src/types/dashboardConfig.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
same as before. I'd leave this warning on, reminding us of things to fix or change in the future

Reply:
I can agree, I left the warning active

Reply:
Here I am leaving the eslint-disable for now, since this file deals with 3rd party objects and we don't have an easy path to avoid non camelcase properties